inst/rmarkdown/templates/jasa/resources/template.tex

```diff
@@ -0,0 +1,172 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%  JASA LaTeX Template File
+%  To make articles using JASA.cls, Version 1.1
+%  September 14, 2019
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%% Step 1:
+%% Uncomment the style that you want to use:
+
+%%%%%%% For Preprint
+%% For manuscript, 12pt, one column style
+
+\documentclass[$for(classoption)$$classoption$$sep$,$endfor$]{JASA}
+
+%%%%% Preprint Options %%%%%
+%% The track changes option allows you to mark changes
+%% and will produce a list of changes, their line number
+%% and page number at the end of the article.
+%\documentclass[preprint,trackchanges]{JASA}
+
+
+%% NumberedRefs is used for numbered bibliography and citations.
+%% Default is Author-Year style.
+%% \documentclass[preprint,NumberedRefs]{JASA}
+
+%%%%%%% For Reprint
+%% For appearance of finished article; 2 columns, 10 pt fonts
+
+% \documentclass[reprint]{JASA}
+
+%%%%% Reprint Options %%%%%
+
+%% For testing to see if author has exceeded page length request, use 12pt option
+%\documentclass[reprint,12pt]{JASA}
+
+
+%% NumberedRefs is used for numbered bibliography and citations.
+%% Default is Author-Year style.
+% \documentclass[reprint,NumberedRefs]{JASA}
+
+%% TurnOnLineNumbers
+%% Make lines be numbered in reprint style:
+% \documentclass[reprint,TurnOnLineNumbers]{JASA}
+
+$if(natbib)$
+\usepackage{natbib}
+$endif$
+
+$if(listings)$
+\usepackage{listings}
+$endif$
+$if(lhs)$
+\lstnewenvironment{code}{\lstset{language=Haskell,basicstyle=\small\ttfamily}}{}
+$endif$
+
+% Pandoc syntax highlighting
+$if(highlighting-macros)$
+$highlighting-macros$
+$endif$
+$if(verbatim-in-note)$
+\usepackage{fancyvrb}
+\VerbatimFootnotes % allows verbatim text in footnotes
+$endif$
+
+$for(header-includes)$
+$header-includes$
+$endfor$
+
+\begin{document}
+%% the square bracket argument will send term to running head in
+%% preprint, or running foot in reprint style.
+
+\title[$if(shorttitle)$$shorttitle$$endif$]{$title$}
+
+% ie
+%\title[JASA/Sample JASA Article]{Sample JASA Article}
+
+%% repeat as needed
+
+$for(author)$
+\author{$author.name$}
+% ie
+%\affiliation{Department1,  University1, City, State ZipCode, Country}
+\affiliation{$author.affiliation$}
+%% for corresponding author
+$if(author.email)$\email{$author.email$}$endif$
+%% for additional information
+$if(author.thanks)$\thanks{$author.thanks$}$endif$
+$endfor$
+
+% ie
+% \author{Author Four}
+% \email{author.four@university.edu}
+% \thanks{Also at Another University, City, State ZipCode, Country.}
+
+%% For preprint only,
+%  optional, if you want want this message to appear in upper left corner of title page
+$if(prerint_notice)$\preprint{$for(author/first)$$author.name$$endfor$, JASA}$endif$
```

Comment:
I think you mean preprint here right ? 
```suggestion
$if(preprint_notice)$\preprint{$for(author/first)$$author.name$$endfor$, JASA}$endif$
```

Also, `$for(author/first)$` is only supported since Pandoc 2.10
https://pandoc.org/releases.html#pandoc-2.10-2020-06-29

We tend to avoid using syntax that would not work on Pandoc 2.7.3 at least (which is the on shipped in RStudio 1.3). 

Using would require making `jasa_article()` only works with Pandoc 2.10+

Reply:
This why the CI is working only with the last Pandoc version

Reply:
5f9cd0f